quodlibet/tests/test_util_collection.py

```diff
@@ -529,9 +532,10 @@ def test_remove_fully(self):
 
 
 class TFileBackedPlaylist(TPlaylist):
```

Comment:
I think `TFileBackedPlaylist` and `TPlaylist` should start with `Test` too (instead of `T`), to be consistent with `TestXPSFBackedPlaylist`

Reply:
Yes, but _that_ discussion... I just counted 299 such classes already in the codebase...

Reply:
I'll change this back then though, it's a fair point about consistency